awspub/configmodels.py

```diff
@@ -94,6 +95,34 @@ class ConfigImageSSMParameterModel(BaseModel):
     )
 
 
+class ConfigSNSNotificationProtocol(str, Enum):
+    DEFAULT = "default"
+    EMAIL = "email"
+
+
+class ConfigImageSNSNotificationModel(BaseModel):
```

Comment:
nit: maybe `ImageSNSNotificationConfig` or `ImageSNSNotificationConfigModel` instead? config at the end reads more clearly (current reading reads like a verb instead of a noun). personal preference is to not repeat "Type" in a subclass unless its intended to be used as a baseclass, but i'm not hard on this.

Reply:
> config at the end reads more clearly

In general I'd agree, but all Models in this file are `Config<something>Model`. We should maintain uniformity here